lib/redmine_pulls/patches/adapters/git_adapter_helper.rb

```diff
@@ -50,7 +50,10 @@ def mergable?(commit_base, commit_head)
             merge_result = nil
             git_cmd(cmd_args) { |io| io.binmode; merge_result = io.read }
 
-            ! (merge_result =~ /<<<<<<<.*=======.*>>>>>>>/m)
+            # Split the regex in to two avoid conflict detection when working with this file
+            regex = Regexp.new("<<<" + "<<<<.*=======.*>>>>>>>", Regexp::MULTILINE)
+
+            ! regex.match(merge_result)
```

Comment:
Can you remove this change from this pull request. I think it was unconsciously commited.